formats/IE.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Description": "IE",
+  "RedundantCharacters": "",
+  "ValidationRegex": "^[AaC-Fc-fHhKkNnPpRrTtV-Yv-y]\\d[0-9Ww][ -]?[0-9AaC-Fc-fHhKkNnPpRrTtV-Yv-y]{4}$",
+  "ValidationRegex.DOC": "https://www.eircode.ie/docs/default-source/Common/prepareyourbusinessforeircode-edition3published.pdf?sfvrsn=2",
+  "TestData": {
+    "Valid": [
+      "D6W1234",
```

Comment:
The regx aint working. I tried it on regexpal
![image](https://user-images.githubusercontent.com/25661023/61368494-98539800-a8ab-11e9-98ae-8bdd64f3c3a2.png)

Reply:
You have to remove the extra `\` in front of `\\d`